site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -178,11 +178,11 @@ the `message GameServer`.
 For language specific documentation, have a look at the respective source (linked above), 
 and the {{< ghlink href="examples" >}}examples{{< /ghlink >}}.
 
-Currently, in [Beta]({{% ref "/docs/Guides/feature-stages.md#feature-gates" %}}), and enabled by default, the current 
-state of the `GameServer` will be sent to the WatchGameServer function on first invocation.  
+{{% feature expiryVersion="1.19.0" %}}
+Currently, in [Beta]({{% ref "/docs/Guides/feature-stages.md#feature-gates" %}}), and enabled by default, the current state of the GameServer will be sent to the WatchGameServer function on first invocation.
```

Comment:
nit: Please leave the backticks around GameServer (for the next week until we delete this block of documentation).